azure-iot-device/azure/iot/device/provisioning/internal/polling_machine.py

```diff
@@ -213,23 +207,21 @@ def _query_operation_status(self, event_data):
         self._operations[rid] = constant.PUBLISH_TOPIC_QUERYING.format(rid, operation_id)
         self._request_response_provider.send_request(
             rid=rid,
-            topic=constant.PUBLISH_TOPIC_QUERYING.format(rid, operation_id),
             request=" ",
```

Comment:
>request=" " [](start = 12, length = 11)

Is this the payload? Why is it always a " "?

Is there some way we could make Request an object that combines this payload with rid to be less confusing as it moves down the stack? I had to do a lot of tracing to realize that the "request" being used in the pipeline is actually this payload.

Reply:
changed the name as of now. Lot of these things will see change eventually.  Things may become operations or events. I am not going to encapsulate as of now.

---
In reply to: [287547668](https://github.com/Azure/azure-iot-sdk-python-preview/pull/97#discussion_r287547668) [](ancestors = 287547668)